PeerioTechnologies/peerio-desktop
# fix(updater): don't show snackbars for newer updates

If there was an update downloaded and ready to install and another
update was discovered, we had multiple snackbars reporting that the
update was downloading and that it will be installed after quitting.

Fix this by not issuing updater snackbars if there's already an update
ready to install.

#### Relevant info and issue/PR links 
 
https://app.clubhouse.io/peerio/story/14047/desktop-updater-shows-multiple-snackbars-if-user-doesn-t-restart

#### Testing instructions  

1. Wait for an optional update to happen. See the snackbars.
2. While the app is running, wait for another update to happen. See no more snackbars.

(Not sure if we should spend the energy on testing step 2, just testing if updates didn't break is okay.)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
